tslint.json

```diff
@@ -13,13 +13,12 @@
     "indent": [ true, "spaces" ],
```

Comment:
Yup - the only departure from what's in denali was removal of the `no-var-require` rule, which didn't make sense to have in here b/c it has to have a couple of requires